app/helpers/application_helper.rb

```diff
@@ -212,13 +211,19 @@ def popover_for_user_name(user, team = nil)
       user_description += %(<p></p></div></div></div>)
     end
 
-    raw(image_tag(Rails.application.routes.url_helpers
-                    .avatar_path(user, :icon_small),
+    raw(image_tag(set_avatar_absolute_url(user, :icon_small),
                   class: 'atwho-user-img-popover')) +
       raw('<a onClick="$(this).popover(\'show\')" ' \
       'class="atwho-user-popover" data-container="body" ' \
       'data-html="true" tabindex="0" data-trigger="focus" ' \
       'data-placement="top" data-toggle="popover" data-content="') +
       raw(user_description) + raw('" >') + user.full_name + raw('</a>')
   end
+
+  def set_avatar_absolute_url(user, style)
```

Comment:
This is a getter not a setter method, but I would name it just `avatar_absolute_url`, in a Ruby spirit.